treeherder/perf/investigation_aids.py

```diff
@@ -0,0 +1,216 @@
+import logging
```

Comment:
investigation_aids.py seems like a too abstract of a name. We can have the taskcluster logic extracted into another module, for everyone to use.